google/cloud/security/inventory/pipelines/load_org_iam_policies_pipeline.py

```diff
@@ -16,64 +16,113 @@
-from google.cloud.security.common.gcp_api import cloud_resource_manager as crm
 from google.cloud.security.common.gcp_api import errors as api_errors
 from google.cloud.security.inventory import errors as inventory_errors
-from google.cloud.security.inventory import transform_util
-
-
-RESOURCE_NAME = 'org_iam_policies'
-RAW_ORG_IAM_POLICIES = 'raw_org_iam_policies'
-
-
-def run(dao=None, cycle_timestamp=None, configs=None, crm_rate_limiter=None):
-    """Runs the load IAM policies data pipeline.
-
-    Args:
-        dao: Data access object.
-        cycle_timestamp: String of timestamp, formatted as YYYYMMDDTHHMMSSZ.
-        configs: Dictionary of configurations.
-        crm_rate_limiter: RateLimiter object for CRM API client.
-
-    Returns:
-        None
-
-    Raises:
-        LoadDataPipelineException: An error with loading data has occurred.
-    """
-    org_id = configs.get('organization_id')
-    # Check if the placeholder is replaced in the config/flag.
-    if org_id == '<organization id>':
-        raise inventory_errors.LoadDataPipelineError(
-            'No organization id is specified.')
-
-    crm_client = crm.CloudResourceManagerClient(rate_limiter=crm_rate_limiter)
-    try:
-        # Retrieve data from GCP.
-        # Flatten the iterator since we will use it twice, and it is faster
-        # than cloning to 2 iterators.
-        iam_policies_map = crm_client.get_org_iam_policies(
-            RESOURCE_NAME, org_id)
-        # TODO: Investigate improving so the pylint disable isn't needed.
-        # pylint: disable=redefined-variable-type
-        iam_policies_map = list(iam_policies_map)
-
-        # Flatten and relationalize data for upload to cloud sql.
-        flattened_iam_policies = (
-            transform_util.flatten_iam_policies(iam_policies_map))
-    except api_errors.ApiExecutionError as e:
-        raise inventory_errors.LoadDataPipelineError(e)
-
-    # Load flattened iam policies into cloud sql.
-    # Load raw iam policies into cloud sql.
-    # A separate table is used to store the raw iam policies because it is
-    # much faster than updating these individually into the projects table.
-    try:
-        dao.load_data(RESOURCE_NAME, cycle_timestamp, flattened_iam_policies)
+from google.cloud.security.inventory.pipelines import base_pipeline
 
+
+class LoadOrgIamPoliciesPipeline(base_pipeline.BasePipeline):
+    """Pipeline to load org IAM policies data into Inventory."""
+
+    RESOURCE_NAME = 'org_iam_policies'
+    RAW_RESOURCE_NAME = 'raw_org_iam_policies'
+
+    def __init__(self, cycle_timestamp, configs, crm_client, dao, parser):
+        """Constructor for the data pipeline.
+
+        Args:
+            cycle_timestamp: String of timestamp, formatted as YYYYMMDDTHHMMSSZ.
+            configs: Dictionary of configurations.
+            crm_client: CRM API client.
+            dao: Data access object.
+            parser: Forseti parser object.
```

Comment:
It's odd to have this passed in and set to an object attribute. Will you ever point this to something other than google.cloud.security.common.util.parser? The import should just be local to this pipeline module and used directly, rather than as self.parser. Same for the other pipelines in this PR.